### PR TITLE
[#32] Get rid of show and show' functions

### LINKED ISSUE #32: Get rid of 'show''
Get rid of `show` and `show'` functions. We have `pretty[L]` in `universum`.

### PR DESCRIPTION
**Pull request by vrom911**
Resolves #32

### CODE REVIEW

File: src/Serokell/Data/Variant/Serialization.hs [hunk 1]
```diff
@@ -33,7 +32,7 @@ import qualified Data.HashMap.Strict as HM
 --    result type depends on sign (negative ⇒ Int, otherwise UInt).
 
 varMapToObject :: VarMap -> Aeson.Object
-varMapToObject = HM.fromList . map (bimap show' Aeson.toJSON) . HM.toList
+varMapToObject = HM.fromList . map (bimap show Aeson.toJSON) . HM.toList
```
Comment (ChShersh) [inline src/Serokell/Data/Variant/Serialization.hs]: Actually, `show` and `show'` from `serokell-util` are `pretty` and `prettyL` from `universum`:

* https://github.com/serokell/universum/blob/b0fc7c1b1ba0aa9c1efba6819d2c7ab0aeea3aae/src/Universum/String/Conversion.hs#L179

File: test/Test/Serokell/Data/Variant/VariantSpec.hs [hunk 2]
```diff
@@ -1,23 +1,25 @@
+{-# LANGUAGE NoImplicitPrelude   #-}
```
Comment (ChShersh) [inline test/Test/Serokell/Data/Variant/VariantSpec.hs]: Maybe it's better to move this into `default-extensions`?

File: test/Test/Serokell/Data/Variant/VariantSpec.hs [hunk 3]
```diff
@@ -58,5 +60,5 @@ jsonMid = maybe err id . A.decode . A.encode
 bytesFun :: S.Variant -> S.Variant
 bytesFun (S.VarString s) = S.VarBytes right
   where
-     right = either (error . unpack) id $ S.decode s
+     right = either error id $ S.decode s
```
Comment (ChShersh) [inline test/Test/Serokell/Data/Variant/VariantSpec.hs]: `either f id` is `first f`. Btw, looks like a good rule to include into `.hlint.yaml` :thinking:

Comment (vrom911) [reply test/Test/Serokell/Data/Variant/VariantSpec.hs]: It's not like that here, `first` is not behaving the same as `either`

File: serokell-util.cabal [hunk 4]
```diff
@@ -126,4 +126,5 @@ test-suite serokell-test
   default-extensions:  OverloadedStrings
                      , RecordWildCards
                      , DeriveDataTypeable
+                     , NoImplicitPrelude
```
Comment (gromakovsky) [inline serokell-util.cabal]: :tada:

File: test/Test/Serokell/Data/Variant/VariantSpec.hs (hunk 3 above)
Comment (ChShersh) [reply test/Test/Serokell/Data/Variant/VariantSpec.hs]: @vrom911 Oh, you're right. Stupid me :man_facepalming:

**Pull request merged**